hw2/src/findPrimeFactor/FindPrimeFactor.java

```diff
@@ -21,17 +20,15 @@ public static int findPrimeFactor (List<Integer> a, List<Integer> b) throws Null
         // Also note that a[] and b[] need not be of the same length.
 
 
-        for (int i = 0; i <= b.size(); i++) {
+        for (int i = 0; i < b.size(); i++) {
```

Comment:
I think this a bug that wasn't caught by the grader but was caught by `result8`.